EngineBay.Authentication/JwtBearerAuth/JwtBearerAuthenticationConfiguration.cs

```diff
@@ -64,7 +64,8 @@ public static IServiceCollection Configure(IServiceCollection services)
                     tokenValidationParameters.TryAllIssuerSigningKeys = false;
                     tokenValidationParameters.SignatureValidator = (token, parameters) =>
                     {
-                        var jwt = new JwtSecurityToken(token);
+                        // var jwt = new JwtSecurityToken(token);
```

Comment:
any reason why we are keeping this line?

Reply:
My bad, forgot to clean it up